src/assets/nav-items.js

```diff
@@ -0,0 +1,40 @@
+const navItems = [
```

Comment:
why you put it in assets file
I prefer to put each js and css file inside component folder